js/components/sortable-densities.js

```diff
@@ -77,11 +78,17 @@ export default class SortableDensities extends Component {
     return (
       <div>
         <div className='densities'>
-          LOW
+          { config.densityWordInPlanetWizard ? 'LOW' : 'ABOVE' }
           <SortableList plateInfos={this.plateInfos} onSortEnd={this.onSortEnd} useDragHandle={false} />
-          HIGH
+          { config.densityWordInPlanetWizard ? 'HIGH' : 'BELOW' }
+        </div>
+        <div className='helper-text'>
+          {
+            config.densityWordInPlanetWizard
+              ? 'Click and drag to reorder the plate density'
+              : 'Click and drag to reorder the plate that will go below the other plate'
```

Comment:
Is this the desired wording? I'd think something like "Click and drag to order the plates from top to bottom" is more clear, especially when there are more than 2 plates

Reply:
I've used wording provided by Amy. I'll double check with her.

Reply:
Amy agreed that this prev version was confusing. She provided another variant that I used in the last commit.